Bottom: b219a4a9def6c76ad83ce9a2d4024378b8b7ad17
Top:    76cf2400d786642b218e61c2ea6620b643e3b6e0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 18:10:18 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index e5c3c17..5bc13b6 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -854,10 +854,10 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
                              -port->view_y0, 0);
 
   glBegin (GL_QUADS);
-  glVertex3i (0, 0, 0);
-  glVertex3i (PCB->MaxWidth, 0, 0);
+  glVertex3i (0,             0,              0);
+  glVertex3i (PCB->MaxWidth, 0,              0);
   glVertex3i (PCB->MaxWidth, PCB->MaxHeight, 0);
-  glVertex3i (0, PCB->MaxHeight, 0);
+  glVertex3i (0,             PCB->MaxHeight, 0);
   glEnd ();
 
   ghid_draw_bg_image ();
